### PR TITLE
backend: remove obsolete TectonicVersion and TectonicCACertFile

### PR DESCRIPTION
**Pull request by spadgett**
/assign @alecmerdler

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -230,7 +229,7 @@ func (s *Server) HTTPHandler() http.Handler {
 		)
 	}
 
-	handle("/api/tectonic/version", authHandler(s.versionHandler))
+	handle("/api/console/version", authHandler(s.versionHandler))
```
Comment (alecmerdler) [inline server/server.go]: This endpoint doesn't exist.

Comment (spadgett) [reply server/server.go]: I don't know what you mean by doesn't exist... Console serves it, although we don't currently use it in the UI.

Comment (alecmerdler) [reply server/server.go]: I tried to access `http://localhost:9000/api/console/version` and get `not found`.

Comment (spadgett) [reply server/server.go]: I changed the path in this pr

Comment (alecmerdler) [reply server/server.go]: Oh sorry, I'm dumb. I thought this was JS code.

**Pull request merged**